core/domain/classifier_services.py

```diff
@@ -127,6 +127,52 @@ def classify_string_classifier_rule(state, normalized_answer):
     return None
 
 
+def create_classifier_training_jobs(exploration, state_names):
+    """Creates ClassifierTrainingJobModel instances for all the state names
+    passed into the function.
+
+    Args:
+        exploration: Exploration. The Exploration Domain object.
+        state_names: list. List of state names.
+    """
+    exp_id = exploration.id
+    exp_version = exploration.version
+    states = exploration.states
+    for state_name in state_names:
+        state = states[state_name]
+        training_data = state.get_training_data()
+        interaction_id = state.interaction.id
+        algorithm_id = feconf.INTERACTION_CLASSIFIER_MAPPING[
+            interaction_id]['algorithm_id']
+        create_classifier_training_job(algorithm_id, interaction_id,
```

Comment:
@seanlip @Arunabh98 . Can I do something along the lines of this? Have a create_multi() function which adds everything to a list and then calls the create_job function for each element in the list? Does that help? Or is that again multiple datastore put()?

Or do you suggest passing in a list of jobs to create and create them all at one go?

What I'm not understanding here is how we do it in one put(). We anyway have to retrieve a job individually and check if it exists, then create it. I think I'm missing something here.